src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -4175,7 +4182,7 @@ void gfx_start_frame() {
                                                     gfx_current_window_dimensions.height, gfx_msaa_level, false, true,
                                                     true, true);
         }
-        if (gfx_msaa_level > 1 && different_size) {
+        if (gfx_msaa_level > 1 && (!viewport_matches_render_resolution())) {
```

Comment:
very much just a nit but
```suggestion
        if (gfx_msaa_level > 1 && !viewport_matches_render_resolution()) {
```